src/Radio Buttons/main.vala

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env -S vala workbench.vala --pkg gtk4
+public void main() {
+    var radio_button_1 = (Gtk.CheckButton) workbench.builder.get_object("radio_button_1");
```

Comment:
Please use spaces before parentheses.
```suggestion
    var radio_button_1 = (Gtk.CheckButton) workbench.builder.get_object ("radio_button_1");
```

Reply:
When I try the code with spaces as requested the workbench itself removes them on running the code. However when I commited the changes the checks failed for the radio button with some formatting error so requesting some suggestions for the same @Diego-Ivan